### PR TITLE
Add tryOnItem() function.

### PR DESCRIPTION
**Pull request by liayoo**

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -140,4 +140,44 @@ export default class Store extends AinftBase {
...
+  }: ItemTryOnParams): Promise<string> {
+    const encodedItemName = encodeURIComponent(itemName);
+    const timestamp = Date.now();
+    const body = {
+      appId,
+      chain,
+      nftContractAddress,
+      nftTokenId,
+    };
+    const data = buildData(
```
Comment (platfowner) [inline src/store.ts:160]: It seems buildData() ~ return axios ... lines are repeated in many places. I think the repeated patterns can be extracted to some member methods of the base class? (For now, some todo would be enough?)

Comment (liayoo) [reply src/store.ts:163]: Thanks, that sounds like a good idea. I left a todo for now.

**Review by woomurf — APPROVED** [review]
LGTM! Thanks!

**Comment by liayoo** [comment]
Thanks for the review, now merging!

**Pull request merged**